include/cadmium/engine/pdevs_coordinator.hpp

```diff
@@ -87,7 +89,12 @@ namespace cadmium {
              * @param t is the start time
              */
             void init(TIME t) noexcept {
-                LOGGER::template log<cadmium::logger::logger_info, decltype(formatter::log_info_init), TIME>(formatter::log_info_init, t);
+
+                //logging data
+                std::ostringstream oss;
+                oss << boost::typeindex::type_id<MODEL<TIME>>().pretty_name();
```

Comment:
I would wrap this in the formatter dependency to avoid dependency in boost::typeindex and ostringstream when not used, for example when working in low resources systems. 
I'm working now in a way to identify models that are named by checking if they have an attribute "pretty_name" and using that attribute in place of the name of the type as name of the model.

Reply:
There is consideration to be made about wrapping this in the formatter, if the MODEL<TIME> type is used whitin the formatter, then this type must be a formatter parameter and there is two problems with that.

1) The compilation performance drops down considerably because there is a different formatter type for each model, but the run time is not improved in any way. 
2) The coordinator must to know how to define the formatter for its sub models, this is not trivial at all.